src/uu/cp/src/cp.rs

```diff
@@ -604,9 +606,19 @@ impl Options {
             || matches.is_present(options::ARCHIVE);
 
         let backup_mode = backup_control::determine_backup_mode(
-            matches.is_present(options::BACKUP_NO_ARG) || matches.is_present(options::BACKUP),
+            matches.is_present(options::BACKUP_NO_ARG),
+            matches.is_present(options::BACKUP),
             matches.value_of(options::BACKUP),
         );
+        let backup_mode = match backup_mode {
+            Err(err) => {
+                return Err(Error::Backup(err));
+                // show_usage_error!("{}", err);
```

Comment:
That's not something you have to address now, but it would be interesting to integrate this with our new error handling mechanism at a later point when all utilities support it, instead of having to repeat the error handling in all utilities.

Reply:
Yes, definitely! I think I'll add it as a `UCommonError`, as it is consistent across utilities.